README-development.md

```diff
@@ -1,3 +1,39 @@
+# Operating System
+
+## Prerequisites
+
+- git
+- coreos-assembler
+- rpmdistro-gitoverlay
+- ostree
+- rpm-ostree 
+- imagefactory-plugins-TinMan
+- qemu-img
+
+
+## Building
+
+- Clone ``openshift/os``
+- Move into the cloned repo
+- Build packages from source repos:
+ - ``rpmdistro-gitoverlay init``
+ - ``rpmdistro-gitoverlay resolve --fetch-all``
+ - ``rpmdistro-gitoverlay build``
+- Make the ostree
+ - ``export REPO=/tmp/repo``
+ - ``ostree init --repo=$REPO --mode=archive --src $REPO/ --dest=$REPO``
```

Comment:
These `--src/--dest` args seem like a copy/paste error from rsync?

Reply:
I actually wasn't 100% sure the right way to convert these lines from the Jenkinsfile without actually going through and testing out to see what happens.

Reply:
It doesn't look like `--src $REPO/ --dest=$REPO` was dropped, definitely should be.

Reply:
It's now removed